ofrak_core/Makefile

```
@@ -18,7 +18,7 @@ test: inspect
 	$(PYTHON) -m pytest -n auto test_ofrak --cov=ofrak --cov-report=term-missing
 	(sleep 2; echo 1; sleep 2; echo i agree) \
 		| python3 -m coverage run --append --source ofrak -m ofrak license --force
-	fun-coverage --cov-fail-under=100
+	if $(PYTHON) -m pip show -qqq ofrak_angr; then fun-coverage --cov-fail-under=100; fi
```

```suggestion
	fun-coverage --cov-fail-under=100
```

The point of fun-coverage here is to assert that all functions are tested. This test should fail if we are skipping tests -- that is a feature, not a bug.

---

@whyitfor , how would one then do a complete regression testing? My go-to approach when I am messing with docker configurations and want to be extra sure I did not break anything is to build both versions (install and develop) of every .yml configuration, and run tests in the develop version of each of them (https://github.com/redballoonsecurity/ofrak/issues/548#issuecomment-2591575019). Does not quite work right now for `ofrak-core-dev.yml` because of python11 incomatibility in some tests, but once that is fixed, wouldn't it be good to retain the ability to test all configurations and have nothing fail?

---

P.S. combined with #572 I just created, this ensures that `make test` in `ofrak-core-dev` develop container succeeds.

---

`make -C ofrak_core test` should pass only if the following are true:

- All of the tests that are run pass
- The tests that are run provide 100% function coverage for the package under test

I guess the implication of this is that it will not pass in an `ofrak-core-dev.yml` container. This is regrettable, but I'm fine with that for now since, as I've mentioned, we will eventually refactor the tests to not rely on disassembler back ends.